### PR TITLE
Html-forms exercise - Heerak Moon

### PR DESCRIPTION
**Pull request by heerakmoon**
https://user-images.githubusercontent.com/113075380/214394252-0f17fcb8-5c9b-47c2-b705-51de1af06b02.mp4

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="index.html">
+      <div>
+        <label for="email">Email</label>
+      </div>
+      <div>
+        <input type="text" name="email">
```
Comment (robgmerrill) [inline html-forms/index.html:13]: The `input` should have an attribute `id` that matches the value for the `label` attribute `for`:

```
<label for="email-address">Email</label>
<div>
  <input name="email" type="email" id="email-address">
</div>
```

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      </div>
+      <div>
+        <label for="reason">Why is coding cool?</label>
+      </div>
+      <div>
+        <select name="reason" id="coding">
```
Comment (robgmerrill) [inline html-forms/index.html:19]: The `id="coding"` should match the `label` elements `for` attribute:
```
<select name="reason" id="reason">
```

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+          <option value="select">Select an option</option>
+          <option value="interesting">Because it's interesting</option>
```
Comment (robgmerrill) [inline html-forms/index.html:21]: For all `'` we want to use html entities: 
`&apos;`

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+          <option value="fun">Because it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <div>
+        <p>Favorite Language (choose one)</p>
+      </div>
+      <div>
+        <input type="radio" name="favorite" value="html">HTML
```
Comment (robgmerrill) [inline html-forms/index.html:30]: `input` tags on line 30, 31, 32 each need their own label.

File: html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+        <input type="radio" name="favorite" value="css">CSS
+        <input type="radio" name="favorite" value="javascript">JavaScript
+      </div>
+      <div>
+        <p>Would you like to sign up for more information?</p>
+      </div>
+      <div>
+        <input type="checkbox" name="signup">
```
Comment (robgmerrill) [inline html-forms/index.html:38]: `input` tag should have an `id` set to the same value as the `label`'s `for` attribute.

**Review by robgmerrill — CHANGES_REQUESTED** [review]
Great start! Few things to work on :)

**Comment by heerakmoon** [comment]
https://user-images.githubusercontent.com/113075380/214425573-f68a984a-a3d2-4173-9566-f2897d2e069b.mp4

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**